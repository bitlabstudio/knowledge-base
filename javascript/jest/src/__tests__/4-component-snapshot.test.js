import React from 'react'
import renderer from 'react-test-renderer'

import TestComponent from '../testedFiles/4-component-snapshot'

describe('TestComponent', () => {
  it('renders the correct values', () => {
    const component = renderer.create(<TestComponent />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
