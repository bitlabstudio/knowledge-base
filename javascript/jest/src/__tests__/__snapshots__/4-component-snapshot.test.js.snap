// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestComponent renders the correct values 1`] = `
<div>
  <h1>
    Test
  </h1>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusantium alias amet asperiores at dicta, dolore doloremque doloribus enim magni nam nesciunt officia quod temporibus unde veritatis! Illo, laudantium, neque. Rerum?
  </p>
</div>
`;
